apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -205,6 +215,29 @@ export default connect(
       }
     }
 
+    displayErrorMsgs() {
```

Comment:
We are referencing the log output in this function as ErrorMsgs, but if I understand correctly, this will be used for things other the displaying errors. Can we rename the errorsMsg variable and this function name to be more consistent with the intended use?

Reply:
Thanks for letting me know! I'll go ahead and change it to `displayOutputToConsole()`.